components/director/internal/domain/formationassignment/repository.go

```diff
@@ -329,8 +330,20 @@ func (r *repository) Update(ctx context.Context, model *model.FormationAssignmen
 	if model == nil {
 		return apperrors.NewInternalError("model can not be empty")
 	}
+	newEntity := r.conv.ToEntity(model)
 
-	return r.updaterGlobal.UpdateSingleGlobal(ctx, r.conv.ToEntity(model))
+	var retrievedEntity Entity
+	if err := r.globalGetter.GetGlobal(ctx, repo.Conditions{repo.NewEqualCondition("id", model.ID)}, repo.NoOrderBy, &retrievedEntity); err != nil {
+		return err
+	}
+
+	if retrievedEntity.State != newEntity.State {
```

Comment:
Here we have the same issue as with the formation entity. The FA can be in `CONFIG_PENDING` and after another wave of notifications it can still be in a `CONFIG_PENDING` but with a completely different `value`. We should consider whether the `value` and `error` fields are changed along with the `state` field.

Reply:
Same as the comment above, I think the timestamp only represents the change in the state not in the response(that is composite of state, error, value)?

Reply:
discussed offline and the code is adjusted.